drivers/src/ecc384.rs

```diff
@@ -170,6 +171,7 @@ impl Ecc384 {
         &mut self,
         seed: Ecc384Seed,
         nonce: &Array4x12,
+        iv: &Array4x12,
```

Comment:
Consider taking a &mut CsRng in as a parameter, and generating the iv here in the driver.

That way, the driver has a stronger guarantee it's actually using random data for the IV, which is necessary for security.

Reply:
Makes sense; updated.